packages/core/src/index.html

```diff
@@ -161,5 +169,123 @@ <h3>Disabled</h3>
         console.log('wayBlur');
       });
     </script>
+
+    <hr>
+    
+    <section>
+      <script>
+        const selectOptions = [
+            { label: 'Arthur Benjamin Dickinson', value: 1001, age: 32 },
+            { label: 'Alexa van der Borre', value: 301, age: 29 },
+            { label: 'Benny Balotelli', value: 2 },
+            { label: 'Eddy Masters', value: 451 },
+            { label: 'Dennis Lei', value: 123 },
+            { label: 'Lynn Kwan', value: 4 },
+            { label: 'Mai Nguyễn', value: 39 },
+            { label: 'Michael Sigþórsson', value: 35 },
+          ];
+      </script>
+      <h2>way-autosuggest</h2>
+      <section>
+        <h3>Basic usage</h3>
+        <way-autosuggest 
+          name="autosuggest"
+          placeholder="Start typing"
+          value='[{ "label": "Benny Balotelli", "value": 2 }]'
+        ></way-autosuggest>
+        <div>
+          <p>autosuggest data:</p>
+          <code class="code-autosuggest-basic"></code>
+          <p>hidden input data:</p>
+          <code class="code-autosuggest-basic-hidden"></code>
+        </div>
+        <script>
+          (function(){
+            const autosuggest = document.querySelector('way-autosuggest');
+    
+            autosuggest.options = selectOptions;
+            autosuggest.config = {
+              maxTags: 6,
+              selectedText: 'selected'
+            }
+            // If value is set in HTML, this js will be ignored.
+            autosuggest.value = [{ label: 'Benny Balotelli', value: 2 }];
+      
+            autosuggest.addEventListener('wayChange', event => {
+              // we get the value both in the event.detail
+              document.querySelector('.code-autosuggest-basic').innerHTML = JSON.stringify(event.detail.value);
+
+              const hiddenInput = document.querySelector('input[name=autosuggest]');
+              if(hiddenInput) {
+                document.querySelector('.code-autosuggest-basic-hidden').innerHTML = hiddenInput.value;
+              }
+            })
+          })()
+        </script>
+      </section>
+      <section>
+        <h3>Prop: options? : Array&lt;WayAutosuggestOption&gt; = []</h3>
```

Comment:
**suggestion**: Nice this documentation! A better place for this would be a docblock comment before the relevant props in the code. Stencil generates a nice markdown doc from this when you run `npm run build` (see the /way-button/readme.me). Which we then can easily copy-paste to our doc site.
The index.html is more like a development playground.